lib/galaxy/util/object_wrapper.py

```diff
@@ -108,10 +106,11 @@
 
 INVALID_CHARACTER = "X"
 
-if sys.version_info > (3, 0):
-    # __coerce__ doesn't do anything under Python anyway.
-    def coerce(x, y):
-        return x
+# __coerce__ doesn't do anything under Python anyway.
+
```

Comment:
empty lines could be removed or removed completely?

Reply:
Yeah, I put the comment inside the function, thank you!